frontend/src/metabase/lib/colors/palette.ts

```diff
@@ -11,7 +11,6 @@ export const ACCENT_COUNT = 8;
 // frontend/src/metabase/css/core/colors.module.css
 // frontend/src/metabase/styled-components/containers/GlobalStyles/GlobalStyles.tsx
 // frontend/src/metabase/styled-components/theme/css-variables.ts
```

Comment:
Certainly, since the work of the SDK, we duplicated these declarations, so now they're a bit confusing, but we should definitely consolidate them. It should be done in a separate PR, since there could be something we couldn't foresee.